test/NuGet.Clients.Tests/NuGet.CommandLine.Test/ConsoleTest.cs

```diff
@@ -17,20 +17,29 @@ public void TestConsoleWindowWidthNotZero()
         }
 
         [Theory]
-        [InlineData(2, "abcd\nwxyz", "  abcdXXX  wxyzXXX", 10)]
-        [InlineData(2, "abcd\rwxyz", "  abcdXXX  wxyzXXX", 10)]
-        [InlineData(3, "abcde\n\nwcjb", "   abcdeXXXXXX   wcjbXXX", 10)]
-        [InlineData(2, "abcde\r\nwcjb", "  abcdeXXX  wcjbXXX", 10)]
-        [InlineData(2, "abcde\rwcjb", "  abcdeXXX  wcjbXXX", 10)]
-        [InlineData(2, "abcdefghijk\nAB", "  abcdeXXX  fghijXXX  kXXX  ABXXX", 7)]
-        [InlineData(2, "ab  cd", "  abXXX  cdXXX", 6)]
-        public void TestPrintJustified(int indent, string input, string expected, int width)
+        [InlineData(2, "abcd\nwxyz", "  abcdXXX  wxyzXXX", 10, 0)]
+        [InlineData(2, "abcd\nwxyz", " abcdXXX wxyzXXX", 9, 1)]
+        [InlineData(2, "abcd\rwxyz", "  abcdXXX  wxyzXXX", 10, 0)]
+        [InlineData(3, "abcde\n\nwcjb", "   abcdeXXXXXX   wcjbXXX", 10, 0)]
+        [InlineData(2, "abcde\r\nwcjb", "  abcdeXXX  wcjbXXX", 10, 0)]
+        [InlineData(2, "abcde\rwcjb", "  abcdeXXX  wcjbXXX", 10, 0)]
+        [InlineData(2, "abcdefghijk\nAB", "  abcdeXXX  fghijXXX  kXXX  ABXXX", 7, 0)]
+        [InlineData(2, "abcdefghijk\nAB", " abcdeXXX fghijXXX kXXX ABXXX", 7, 1)]
+        [InlineData(2, "ab  cd", "  abXXX  cdXXX", 6, 0)]
```

Comment:
Does this one work? Thinking perhaps the "e" never gets to the output. Also wasn't sure if we expected the space in the output at all, so adjust my expected value depending on that ("abcdXXX e" ?)
```suggestion
        [InlineData(2, "ab  cd", "  abXXX  cdXXX", 6, 0)]
        [InlineData(0, "abcd e", "abcdXXXe", 5, 0)]
```

Reply:
Yes, it works. I adapted your test slightly to include the newline (XXX) at the end of the expected output too. So I ran: `[InlineData(0, "abcd e", "abcdXXXeXXX", 5, 0)]` and it passed. I've added this test in the latest commit as well.